### PR TITLE
docs: multi-agent docs

### PR DESCRIPTION
**Pull request by kevinthecheung**
Description here... 

Checklist (if applicable):
- [ ] Tested (manually, unit tested, etc.)
- [ ] Docs updated

### CODE REVIEW

File: docs/multi-agent.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+    tools: [reservationAgent, menuInfoAgent, complaintAgent],
+  },
+  `{{role "system"}} You are an AI customer service agent for Pavel's Cafe.
+  Greet the user and ask them how you can help. If appropriate, transfer to an
+  agent that can better handle the request. If you cannot help the customer with
+  the available tools, politely explain so.`
+);
+
+// Start a chat session, initially with the triage agent
+const chat = ai.chat({ prompt: triageAgent });
```
Comment (pavelgj) [inline docs/multi-agent.md:113]:
```suggestion
const chat = ai.chat({ preamble: triageAgent });
```

**Comment by i14h** [comment]
Nice!

File: docs/multi-agent.md [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+const menuLookupTool = ai.defineTool(
+  {
+    name: 'menuLookupTool',
+    description: 'use this tool to look up the menu for a given date',
+    inputSchema: z.object({
+      date: z.string().describe('the date to look up the menu for'),
+    }),
+    outputSchema: z.string().describe('the menu for a given date'),
+  },
+  async (input): Promise<string> => {
```
Comment (pavelgj) [inline docs/multi-agent.md:22]: should be inferred...

```suggestion
  async (input) => {
```

File: docs/multi-agent.md [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+      date: z.string().describe('the date to book for'),
+    }),
+    outputSchema: z
+      .string()
+      .describe(
+        "true if the reservation was successfully booked and false if there's" +
+        " no table available for the requested time"
+      ),
+  },
+  async (input): Promise<string> => {
```
Comment (pavelgj) [inline docs/multi-agent.md:42]:
```suggestion
  async (input) => {
```

**Review by pavelgj — APPROVED** [review]

**Pull request merged**